healthcareai/common/helpers.py

```diff
@@ -1,6 +1,8 @@
 import math
 import sklearn
 
```

Comment:
Wasn't from this change, but would we only want to import the functionality needed (and not the whole package)?

Reply:
https://softwareengineering.stackexchange.com/questions/187403/import-module-vs-from-module-import-function